sql/core/src/test/scala/org/apache/spark/sql/ExplainSuite.scala

```diff
@@ -527,6 +527,20 @@ class ExplainSuite extends ExplainSuiteHelper with DisableAdaptiveExecutionSuite
 class ExplainSuiteAE extends ExplainSuiteHelper with EnableAdaptiveExecutionSuite {
   import testImplicits._
 
+  /**
+   * Get the final plan explain after running the sql
+   */
+  private def withFinalPlanNormalizedExplain(query: String, mode: String)(f: String => Unit) = {
```

Comment:
Cannot we reuse `withNormalizedExplain`? Seems you can pass a DataFrame for the query?

Reply:
we can, just missed that method